### DISCUSSION TITLE
how to change vue-cli-service build lib image address in css to absolute path

### DISCUSSION THREAD

**Discussion by arishojaei** [vuejs/vue-cli#7103] [Q&A]
I'll make my question short!

I write this in my `*.scss` file:
```css
background-image: url("~@/assets/images/Logo-en.svg");
```

after I run `yarn build` I get this in my `*.css`:
```css
background-image:url(/img/Logo-fa.ad1b8dbc.svg)
```

***BUT***, I don't want first `/` in url:
```css
background-image:url(img/Logo-fa.ad1b8dbc.svg)
```

I use `vue-cli-serice` to build a component library and when I use this library in a Nuxtjs project, my images didn't load because images address are relative and didn't read from `/dist/img/` in my *package* in my n*ode_modules*

**Comment by Viggo95** [comment]
Can you provide a mini repo for reproduction?

**Comment by Viggo95** [comment]
@arishojaei 

If `@vue/cli-service` is 4.x

You can try this code and change it in your way 👇
```js
module.exports = {
  chainWebpack(config => {
    config.module
      .rule('svg')
      .use('file-loader')
      .tap(opts => ({
        ...opts,
        publicPath: process.cwd(),
      }))
  })
}

```

<br>

If `@vue/cli-service` is 5.x
```js
module.exports = {
  chainWebpack(config => {
    config.module
      .rule('svg')
      .set('generator', {
        publicPath: process.cwd()
      })
  })
}
```


Then, build your component library
